src/resources/i18n/translations/en.json

```
@@ -419,42 +418,42 @@
 	"No summary available": "No summary available.",
 	"No thanks": "No thanks",
 	"No": "No",
-	"Now Playing Info": "\"Now playing\" info",
-	"Now Playing": "Now playing",
+	"Now Playing Info": "\"Now Playing\" Info",
+	"Now Playing": "Now Playing",
 	"OFFLINE MODE ENABLED": "Offline mode on",
 	"OK": "OK",
-	"Official Episode Page": "Official episode page",
-	"Official Podcast Page": "Official podcast page",
-	"Offline Mode": "Offline mode",
+	"Official Episode Page": "Official Episode Page",
+	"Official Podcast Page": "Official Podcast Page",
+	"Offline Mode": "Offline Mode",
 	"Offline mode can save battery life and improve performance": "Can save battery life and improve performance in areas with poor connectivity.",
 	"Ok": "OK",
-	"Only allow downloading episodes when connected to Wifi": "Wi-Fi only",
-	"Only with Link means only people who": "\"Only with link\" means it not in the \"Public\" list on Podverse and the clip's link is needed to play it. Create public clips with Podverse premium.",
+	"Only allow downloading episodes when connected to Wifi": "Only allow downloading episodes when connected to Wifi",
```

```suggestion
	"Only allow downloading episodes when connected to Wifi": "Only allow downloading episodes when connected to Wi-Fi",
```
Isn't it also for clips?
If I care not to download over the cellular network, I don't really care _what_ it is that is being downloaded.
With "allow" it will now wrap across two lines.
It is right below "Downloads", so "downloading isn't needed". "Only" is carried over.
"Only on Wi-Fi" is used in F-Droid, Droidify, etc.

---

The reason for downloading _episodes_ specifically over WiFi is because they are the largest files. I've used this this setting implemented like this in other podcast apps.

If people want the current setting to behave differently, we'll need to create it as a new feature, and would probably add it as a separate toggle in Settings.

That's a great point about "Downloads" carrying over. I'd prefer the "Only on Wi-Fi" phrasing in this case instead of "Wi-Fi only".